nautobot/circuits/forms.py

```diff
@@ -315,12 +315,13 @@ class CircuitFilterForm(BootstrapMixin, TenancyFilterForm, StatusFilterFormMixin
 #
 
 
-class CircuitTerminationForm(BootstrapMixin, RelationshipModelForm):
+class CircuitTerminationForm(BootstrapMixin, RelationshipModelForm, CustomFieldModelForm):
```

Comment:
Let's stay aware of the timing of this PR versus #1364 - if we need to do a followup commit to make sure this form is taking advantage of that enhancement, so be it - core team can certainly handle that. :-)

Reply:
Sure! I wouldn't mind assisting as long as the timing is good.

Reply:
#1364 is merged now - would you mind updating this line accordingly? Also please change the `mdi-cog` icon below to `mdi-information-outline`. Thank you!